ix-dev/stable/nextcloud/templates/docker-compose.yaml

```
@@ -174,6 +174,13 @@
   {% do cron_container.environment.add_user_envs(values.nextcloud.additional_envs) %}
 {% endif %}
 
+{% if values.nextcloud.imaginary.enabled %}
+  {% set imaginary_container = tpl.add_container(values.consts.imaginary_container_name, "imaginary_image") %}
+  {% do imaginary_container.add_caps(["SYS_NICE"]) %}
```

```suggestion
  {% do imaginary_container.set_user(65534, 65534) %}
  {% do imaginary_container.add_caps(["SYS_NICE"]) %}
```

Now I'm thinking about this, given this is usually the nobody user, it might also work with 568.
Can you give it a shot?